Cargo.toml

```diff
@@ -226,3 +226,22 @@ substrate-bn = { git = "https://github.com/sp1-patches/bn", tag = "patch-0.6.0-s
 sha3 = { git = "https://github.com/sp1-patches/RustCrypto-hashes", package = "sha3", tag = "patch-sha3-0.10.8-sp1-4.0.0" }
 p256 = { git = "https://github.com/sp1-patches/elliptic-curves", tag = "patch-p256-13.2-sp1-5.0.0" }
 k256 = { git = "https://github.com/sp1-patches/elliptic-curves", tag = "patch-k256-13.4-sp1-5.0.0" }
+
+
+[patch."https://github.com/op-rs/kona"]
+kona-mpt = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-derive = { git = "https://github.com/celo-org/kona", rev = "bafc266", default-features = false }
+kona-driver = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-preimage = { git = "https://github.com/celo-org/kona", rev = "bafc266", features = [
+    "rkyv",
+    "serde",
+] }
+kona-executor = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-proof = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-client = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-host = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-providers-alloy = { git = "https://github.com/celo-org/kona", rev = "bafc266" }
+kona-rpc = { git = "https://github.com/celo-org/kona", rev = "bafc266", default-features = false }
+kona-protocol = { git = "https://github.com/celo-org/kona", rev = "bafc266", default-features = false }
+kona-registry = { git = "https://github.com/celo-org/kona", rev = "bafc266", default-features = false }
+kona-genesis = { git = "https://github.com/celo-org/kona", rev = "bafc266", default-features = false }
```

Comment:
Shouldn't this be referencing https://github.com/celo-org/kona/commit/d6b2bd3 (the same commit that was referenced by the celo-kona PR?